### DISCUSSION TITLE
Binding ListBox selectedItems to a collection in the viewModel

### DISCUSSION THREAD

**Discussion by brignolff** [reactiveui/ReactiveUI#3382] [Q&A]
Hi,

I'm trying to bind a list box selected items to a collection in my ViewModel and then validate the collection using a ValidationRule.
The ValidationRule only verify if the count property of the collection is higher than 0.

The collection is getting filled but the Validation rule doesn't re-evaluate.

Tried with sourceList/Caches and Observable collection.

Can someone help me ?

**Comment by mStylias** [comment]
Hey @brignolff ,
[This video](https://youtu.be/o4Yoo6EQ_YY?list=PLFJQnCcZXWjtQOb-V44yS0FnUHsccS32n) explains Validation in ReactiveUI pretty well. Maybe it will help you.

**Reply by brignolff** [reply]
Thank you for your answer, I finally created an observable of IChangeSet using the connect method of the sourcelist. I then used it to trigger the validation rule.
